CMakeLists.txt

```diff
@@ -61,6 +61,8 @@ if(NOT CMAKE_BUILD_TYPE)
   set_property(CACHE CMAKE_BUILD_TYPE PROPERTY STRINGS "Debug" "Release" "MinSizeRel" "RelWithDebInfo")
 endif()
 
+set(CMAKE_CXX_STANDARD 17)
+
```

Comment:
you need to apply the c++17 also on windows side

Reply:
Will do!